server/src/routes/Authorize.ts

```diff
@@ -52,6 +52,7 @@ export async function authorizationCallback(req: Request, res: Response) {
         * to display or store the error
          */
         console.log(e);
+        logger.info(e)
```

Comment:
If we want to start using the logger, we should probably move away from the console logs. Also in this case it should be `logger.error(e)`.